examples/with-jest-react-testing-library/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "with-jest-react-testing-library",
+  "version": "1.0.0",
+  "dependencies": {
+    "next": "^7.0.2",
+    "react": "16.7.0-alpha.2",
+    "react-dom": "16.7.0-alpha.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.1.2",
```

Comment:
Is this needed? You'd override the version that is installed with Next.js

Reply:
It is not needed, thanks for pointing out. :)

Reply:
💯